### ISSUE TITLE
[EPIC] pfe-navigation 1.x | Minor bug fixes and small features

### ISSUE THREAD

**Issue by castastrophe** [patternfly/patternfly-elements#1637]
#### Styles
- [ ] line height bug on menu items ([#1454](https://github.com/patternfly/patternfly-elements/issues/1454))
- [ ] add min-width to mobile menu icon ([#781](https://github.com/patternfly/patternfly-elements/issues/781))
- [ ] autoprefixer warnings ([#1214](https://github.com/patternfly/patternfly-elements/issues/1214))
- [ ] position of caret on mobile menu item with no tray ([#795](https://github.com/patternfly/patternfly-elements/issues/795))

#### JS
- [ ]  doesn't work with Angular ([#919](https://github.com/patternfly/patternfly-elements/issues/919))
  - [ ] Update tests to use new tooling; will make validating Angular easier
- [ ]  doesn't render slot="mobile-login" unless slot"mobile-language" exists ([#1106](https://github.com/patternfly/patternfly-elements/issues/1106))
- [ ]  links inside pfe-modal inside pfe-navigation don't work ([#873](https://github.com/patternfly/patternfly-elements/issues/873))

#### Styles
- [ ]  update styles to use unprefixed attributes ([#1501](https://github.com/patternfly/patternfly-elements/issues/1501))
- [ ]  ability to remove heading tags in pfe-navigation without visual changes ([#773](https://github.com/patternfly/patternfly-elements/issues/773))

#### JS
- [ ]  mobile links need event hooks ([#945](https://github.com/patternfly/patternfly-elements/issues/945))

### Possible but could be too large to include

- [ ]  should have a general purpose, utility slot for secondary links ([#759](https://github.com/patternfly/patternfly-elements/issues/759))

**Comment by castastrophe** [comment]
[US295627](https://rally1.rallydev.com/#/270861059696d/teamboard?Name=PFE%20%7C%20pfe-navigation%201.x%20bug%20fixes%20and%20improvements&Owner=%2Fuser%2F16404431353&Project=%2Fproject%2F270861059696&detail=%2Fuserstory%2F600003110815&iteration=u&rankTo=BOTTOM&view=113b0e72-0781-41c7-b6d6-e161d3349c72&fdp=true?fdp=true): PFE | pfe-navigation 1.x bug fixes and improvements

**Comment by castastrophe** [comment]
@bennypowers I suspect you can close this one :)
